tooling/version.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "Build Packages and Script"
+npm run build
+npm run build:script
+
+echo "Bump all package version"
+npm run tooling -- version set "${NPM_PACKAGE_VERSION}" --all
+
+echo "Generate Changelog"
+npx conventional-changelog-cli -p angular -i CHANGELOG.md -s
```

Comment:
ignore this npx for now i was just using it to test it out before adding it to the package.json